gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/PortalRedirectResource.java

```diff
@@ -73,7 +75,15 @@ public Response redirectToPortal(@Context final HttpServletRequest httpServletRe
                     .build();
                 url = uriComponents.toUriString();
             }
-            var versionQueryParam = version != null && version.equals(PORTAL_NEXT_VERSION) ? PORTAL_NEXT_QUERY_PARAM : "";
+            var versionQueryParam = "";
+            if (version != null) {
+                if (version.equals(PORTAL_NEXT_VERSION)) {
+                    versionQueryParam = PORTAL_NEXT_QUERY_PARAM;
+                } else if (version.equals(PORTAL_CLASSIC_VERSION)) {
+                    versionQueryParam = PORTAL_CLASSIC_QUERY_PARAM;
+                }
+            }
```

Comment:
this can be a bit simplified


```suggestion
            var versionQueryParam = switch(version) {
              case PORTAL_NEXT_VERSION -> PORTAL_NEXT_QUERY_PARAM;
              case PORTAL_CLASSIC_VERSION -> PORTAL_CLASSIC_QUERY_PARAM;
              case null, default -> "";
            };
```